interface-definitions/include/openconnect-config-per-x.xml.i

```diff
@@ -0,0 +1,54 @@
+<!-- include start from openconnect-config-per-x.xml.i -->
+<node name="config-per-x">
+    <properties>
+        <help>Configures ocserv to search the configured directory for a config file matching the Group name or username</help>
```

Comment:
The fact that the daemon is named "ocserv" is an implementation detail. I believe in user-visible parts, it should always be "OpenConnect".